Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo

```diff
@@ -1663,35 +1676,35 @@ This model contains the mechanical components of the r3 robot
               extent={{-200,250},{200,210}},
               textString="%name",
               lineColor={0,0,255}),
+            Bitmap(extent={{-130,-195},{195,195}},
+              fileName="modelica://Modelica/Resources/Images/Mechanics/MultiBody/Examples/Systems/robot_kr15.png"),
             Text(
-              extent={{-200,-150},{-140,-190}},
+              extent={{-190,-130},{-130,-170}},
               textString="1",
-              lineColor={0,0,255}),
+              lineColor={0,0,0}),
             Text(
-              extent={{-200,-30},{-140,-70}},
-              textString="3",
-              lineColor={0,0,255}),
-            Text(
-              extent={{-200,-90},{-140,-130}},
+              extent={{-190,-70},{-130,-110}},
               textString="2",
-              lineColor={0,0,255}),
+              lineColor={0,0,0}),
             Text(
-              extent={{-200,90},{-140,50}},
-              textString="5",
-              lineColor={0,0,255}),
+              extent={{-190,-10},{-130,-50}},
+              textString="3",
+              lineColor={0,0,0}),
             Text(
-              extent={{-200,28},{-140,-12}},
+              extent={{-190,48},{-130,8}},
               textString="4",
-              lineColor={0,0,255}),
+              lineColor={0,0,0}),
             Text(
-              extent={{-198,150},{-138,110}},
+              extent={{-190,110},{-130,70}},
+              textString="5",
+              lineColor={0,0,0}),
+            Text(
+              extent={{-188,170},{-128,130}},
```

Comment:
The difference is small enough that it shouldn't matter. But just for consistency with others shouldn't it be {{-190,170},{-130,130}}?